docs/en/anticorona_drones.md

```diff
@@ -70,16 +94,30 @@ Detecting people without masks is cool, no doubt.
-
-<img src="../assets/anticorona/release_Cad.jpg" title="Mask release manufactured" width=300 class="zoom center">
+<p align="center">
+  <img src="../assets/anticorona/release.jpg" title="Mask release in cad" width=300 class="zoom center">
+&nbsp; &nbsp;
+  <img src="../assets/anticorona/release_Cad.jpg" title="Mask release manufactured" width=300 class="zoom center">
+</p>
 
 This device looks like a regular firearm mag, and functions exactly in the same way. Masks can be loaded into containers made out of 20ml syringe barrels.
 
-This device needs further engineering because current iterations are too fragile and unreliable, probably the best solution will be to use linear actuator and push the "casing" out of the action, like in actual firearm.
+This device needs further engineering because current iterations are too fragile and unreliable, probably the best solution will be to use linear actuator and push the "casing" out of the action, like in actual firearm. 
 
 ## Final thoughts
+We all hope that Corona crisis will soon be over, and when it will be finally over, our drones will be still useful. We can deliver some small objects, like cosmetics or shaving blades to the customers' door, the task that currently is done by a human courier. This service (with shaving blades), when a guy comes and brings a new set of shaving blades every week is very popular in US and UK, so why not try to automate it. 
+
+This is an MVP, so some improvements are to be done. For example, payload refill and battery swaps are done manually for now, but this task should be automated. Actually, some work is already done in this direction:
 
-This project will still be useful even when the Coronavirus crisis will be over. It can deliver some small objects, like cosmetics or shaving blades to the customers' door, the task that currently is done by a human courier. This service (with shaving blades), when a guy comes and brings a new set of shaving blades every week is popular in US and UK, so why not try to automate it a bit. The working principle and hardware will remain the same, but software will need an update.
+<img src="../assets/anticorona/charging_drone.jpg" title="Mask release manufactured" width=300 class="zoom center">
+
+Drone lands on a landing platform with special hooks which will connect to four metallic contacts (highlighted) and charge the battery. In future we want to change battery, not just charge it, but the mechanism is very complex and requires making custom batteries and battery mounts, also, it constraints placement of cameras, payload and so on.
+
+For the general-purpose delivery drone, the working principle and hardware will remain the same, but software (neural net detector) will need an update.
 
 Stay safe folks!
+
+## References
+> \[1\] https://www.researchgate.net/publication/348093195_Unmanned_Aerial_Vehicle_Path_Planning_for_Exploration_Mapping
```

Comment:
This is not the syntax for references, see https://raw.githubusercontent.com/CopterExpress/clover/master/docs/en/dromap.md.

Reply:
Resolved